Plash/Utilities.swift

```diff
@@ -1679,19 +1679,32 @@ extension WKWebView {
 	// TODO: Use https://developer.apple.com/documentation/webkit/wkwebview/3516410-mediatype when macOS 10.15.4 is out.
 	// TODO: Move this to `SSWebView` instead and also expose a `response` property so we don't need the `mimeType` parameter.
 	// https://github.com/feedback-assistant/reports/issues/82
-	/// Center a standalone image as WKWebView doesn't center it like Chrome and Firefox do...
-	func centerImage(mimeType: String?) {
+	/**
+	Centers a standalone image as WKWebView doesn't center it like Chrome and Firefox do.
+	The image will aspect-fill the space available.
+
+	You can test this functionality at https://codepen.io/colejd/pen/MWyxezd
```

Comment:
I think you misunderstood the request. I was asking for a URL that can be put into Plash to test that Plash does the correct thing. In hindsight, I guess we cannot use CodePen for this as it doesn't let us show the full page. Not important enough to spend more time on this.